draft-high-assurance-profile-oid4vc-sd-jwt-vc.md

```diff
@@ -176,40 +185,36 @@ All the requirements defined in [@!I-D.ietf-oauth-selective-disclosure-jwt] MUST
+Note: If there is a requirement to provide the Subject’s identifier assigned and maintained by the Issuer, `sub` claim MAY be used. There is no requirement for a binding to exist between `sub` and `cnf` claims. See section X in @!I-D.draft-terbu-sd-jwt-vc for implementation considerations.
 
-Note: The issuer MAY decide to support both options. In which case, it is at the discretion of the Wallet and the Verifier which key to use for the issuer signature validation.
+Note: In some credential types, it is not desirable to include an expiration date (eg: diploma attestation). Therefore, this profile leaves its inclusion to the Issuer, or the body defining the respective credential type.
 
-## Holder key resolution {#holder-key-resolution}
+## Issuer identification and key resolution to validate an issued Credential {#issuer-key-resolution}
 
-* The public key passed in the `cnf` claim [@!RFC7800] MUST be a JSON Web Key [@!RFC7517] contained in the `jwk` claim. `kid' MUST be included in the JWK in the `cnf` claim, since it will be used in the JOSE Header.
+This profile supports two ways to represent and resolves the key required to validate the issuer signature of a SD-JWT-VC, web-based key resolution and x.509 certificates.
 
-Note: this requirement might change when claim-based and biometrics-based holder binding become available.
+* Web-based key resolution: The key used to validate the Issuer’s signature on the VC-SD-JWT MUST be obtained from the VC-SD-JWT issuer's metadata as defined in Section 5 of [!I-D.ietf-terbu-sd-jwt-vc]. The JOSE header `kid` MUST be used to identify the respective key.
+* x.509 certificates: the VC-SD-JWT contains the issuer's certificate along with a trust chain in the `x5c` JOSE header. In this case, the `iss` value MUST be an URL with a FQDN matching a `dNSName` Subject Alternative Name (SAN) [@!RFC5280] entry in the leaf certificate.
 
-Note: Re-using the same Credential across Verifiers, or re-using the same JWK value across multiple Credentials gives colluding Verifiers a mechanism to correlate the User. There are currently two known ways to address this with SD-JWT VCs. First is to issue multiple instances of the same credentials with different JWK values, so that if each instance of the credential is used at only one Verifier, it can be reused multiple times. Another is to use each credential only once (ephemeral credentials). It is RECOMMENDED to adopt one of these mechanisms.
+Note: The issuer MAY decide to support both options. In which case, it is at the discretion of the Wallet and the Verifier which key to use for the issuer signature validation.
 
 ### Cryptographic Holder Binding between VC and VP
 
-* Holder Binding JWT MUST be supported as defined in [@!I-D.ietf-oauth-selective-disclosure-jwt].
+* For Cryptographic Holder Binding, an HB-JWT as defined in [!I-D.ietf-terbu-sd-jwt-vc] MUST always be present when presenting a VC-SD-JWT.
```

Comment:
I think we can remove this section (lines 215-217) since now HB JWT is defined in SD-JWT spec itself and oliver's draft mandates HB jwt?

Reply:
oh wow, I thought we have agreed to make HB-JWT mandatory in sd-jwt vc. opened an [issue #110](https://github.com/vcstuff/draft-terbu-sd-jwt-vc/issues/110) there and here - issue #38